code/game/objects/items/weapons/dice.dm

```diff
@@ -326,11 +326,13 @@
 /datum/context_click/digi_die/action(obj/item/used_item, mob/usr, params)
 	var/obj/item/weapon/dice/borg/d = holder
 	if(return_clicked_id_by_params(params))
-		d.set_sides()
+		d.set_sides(usr)
 		return 1
 
-/obj/item/weapon/dice/borg/proc/set_sides()
+/obj/item/weapon/dice/borg/proc/set_sides(mob/usr)
```

Comment:
Do I correct every instance of usr seen in this file as well? I remember some sort of crusade against either usr or user but I couldn't find which.
I'm testing each revision before it's pushed so thats something at least.